src/components/Select/Select.test.tsx

```diff
@@ -365,7 +366,9 @@ describe('Multi Select', () => {
 
   it('selects all options when Select All is clicked', async () => {
     userEvent.click(selectInput);
-    userEvent.click(screen.getByTestId('ictinus_list_default_option'));
+    userEvent.click(screen.getByTestId(`ictinus_list_item_${SELECT_ALL_OPTION.value}`));
+
+    screen.debug();
```

Comment:
is this necessary?

Reply:
solved at [d91f92ff85eb85f7510d3d67ca3e0130ec88fdce](https://github.com/Orfium/orfium-ictinus/pull/655/commits/d91f92ff85eb85f7510d3d67ca3e0130ec88fdce)